en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-graphql-api-in-choreo-connect.md

```diff
@@ -0,0 +1,296 @@
+    - `graphql-complexity.yaml` is an optional file where you can define complexities relevant to the GraphQL queries.
+
+### Step 2 - Add a Choreo Connect Environment to apictl
+
+To use apictl with Choreo Connect, a Choreo Connect environment needs to be added to the apictl. This environment will hold the adapter URL for further commands.
+
+``` bash
+apictl mg add env dev --adapter https://localhost:9843
+```
+
+!!! info
+
+    Note `mg` in the above command. The apictl commands that starts as `apictl mg` are Choreo Connect specific. If a command does not have `mg` after `apictl` then the command could probably be common to both Choreo Connect and API Manager, but it could also be API Manager specific. 
+
+!!! tip
+
+    The apictl commands here onwards are executed with the -k flag to avoid SSL verification with the Choreo Connect.
+
+    To communicate via HTTPS without skipping SSL verification (without -k flag), add the cert in the Choreo Connect truststore into the `<USER_HOME>/.wso2apictl/certs` folder.
+
+### Step 3 - Log in to the Choreo Connect Environment in apictl
+
+You can use the following command to log in to the above Choreo Connect cluster (log in to the Choreo Connect adapter). When you log in, an access token will be retrieved from Choreo Connect and it will be saved in  the apictl.
+
+``` bash tab="Format"
+apictl mg login dev -u <username> -p <password> -k
+```
+
+``` bash tab="Example"
+apictl mg login dev -u admin -p admin -k
+```
+
+### Step 4 - Deploy the API
+
+Now let's deploy the API to Choreo Connect by executing the following command.     
+
+``` bash
+apictl mg deploy api -f <path_to_the_API_project_just_created>/<gqlProjectName> -e dev -k
+```  
+
+### Step 5 - Generate an access token
+
+After exposing the GraphQL API via Choreo Connect, you can invoke an API with a valid access token.
+
+Let's use the following command to generate a JWT to access the API, and set it to the variable `TOKEN`. 
+
+``` tab="Format"
+TOKEN=$(curl -X POST "https://<hostname>:<port>/testkey" -d "scope=<scope values>" -H "Authorization: Basic base64(username:password)" -k -v)
+
+```
+
+``` tab="Example"
+TOKEN=$(curl -X POST "https://localhost:9095/testkey" -d "" -H "Authorization: Basic YWRtaW46YWRtaW4=" -k -v)
+
+```
+
+
+### Step 6 - Invoke the API
+
+!!! note
+    - Choreo Connect handles GraphQL API requests considering the `POST` HTTP method type only. Therefore, the query given in the request should follow the JSON structure explained in [here](https://graphql.org/learn/serving-over-http/#post-request).
+    - Request should include one of the http request `Content-Type` headers listed in below. 
+        - `-H "Content-Type: application/json"`
+        - `-H "Content-Type: application/graphql"`
+
+
+1. Invoke the API using the following cURL command structure.
```

Comment:
Change this to a bullet point. We use numbers if there is some sequence and in this case there is only one item in the list so a bullet is more appropriate.,